.github/workflows/main.yaml

```diff
@@ -189,6 +233,65 @@ jobs:
           type: ondevice
           os: linux
 
+  # Runs on-host integration and unit tests.
+  on-device-test:
+    needs: [initialize, build]
+    if: |
+      needs.initialize.outputs.on_device_test == 'true' && (
+        inputs.nightly ||
+        inputs.postsubmit ||
+        contains(github.event.pull_request.labels.*.name, 'on_device')
+      )
+    runs-on: [self-hosted, linux, X64]
+    name: On Device Test (${{ matrix.config }}/${{ matrix.type }} on ${{ matrix.platform }})
```

Comment:
Can we change name for on device test jobs, otherwise they get truncated (see https://screenshot.googleplex.com/9d6Tyd82XzKH4Eo.png). Maybe something like what we do for on the host test, something like so:

`name: ${{matrix.name}}_${{matrix.type}}`

Reply:
resolved